go.mod

```diff
@@ -11,20 +11,23 @@ require (
 	github.com/containerd/go-runc v0.0.0-20190923131748-a2952bc25f51 // indirect
 	github.com/containerd/project v0.0.0-20190513184420-7fb81da5e663
 	github.com/containerd/ttrpc v0.0.0-20190828154514-0e0f228740de
-	github.com/containernetworking/cni v0.7.1
-	github.com/containernetworking/plugins v0.8.3
-	github.com/containers/buildah v1.11.5-0.20191031204705-20e92ffe0982
+	github.com/containernetworking/cni v0.7.2-0.20200304161608-4fae32b84921
+	github.com/containernetworking/plugins v0.8.5
+	github.com/containers/buildah v1.14.2
 	github.com/containers/conmon v2.0.11+incompatible
 	github.com/containers/image/v5 v5.2.1
-	github.com/containers/libpod v1.6.3-0.20191101152258-04e8bf3dba50
+	github.com/containers/libpod v1.8.1
```

Comment:
1.8.2 just released yesterday, I'm not sure if you want to bump this now or not.